dcos_installer/cli.py

```diff
@@ -136,7 +136,7 @@ def do_validate_config(args):
     'genconf': (
         lambda args: backend.do_configure(),
         'EXECUTING CONFIGURATION GENERATION',
-        'Execute the configuration generation (genconf).'),
+        'Create customized DC/OS build files for the cluster in {}.'.format(dcos_installer.constants.SERVE_DIR)),
```

Comment:
That works for me. I copied that line nearly verbatim from: https://docs.mesosphere.com/1.9/administration/installing/custom/cli/ (bullet 1 under "To install DC/OS")